aspnet/security/data-protection/compatibility/cookie-sharing.rst

```diff
@@ -10,18 +10,22 @@ Sharing authentication cookies between applications
 
 To share authentication cookies between two different ASP.NET Core applications, configure each application that should share cookies as follows.
 
-1. Add Authentication to your app
+1. In your configure method use the CookieAuthenticationOptions to set up the data protection service for cookies and the AuthenticationScheme to match ASP.NET 4.X.
```

Comment:
Looks like this is hte only step in this section, so I think this numbering can be removed. There are some .rst formatting issues below as well. I'll do a quick pass to clean this up.